extensions/ql-vscode/src/cli.ts

```diff
@@ -613,6 +613,21 @@ export class CodeQLCliServer implements Disposable {
 
     return await this.runCodeQlCliCommand(['database', 'unbundle'], subcommandArgs, `Extracting ${archivePath} to directory ${target}`);
   }
+  
+  /**
+   * Uses a .qhelp file to generate Query Help documentation in a specified format.
+   * @param pathToQhelp The path to the .qhelp file
+   * @param format The format in which the query help should be generated {@link https://codeql.github.com/docs/codeql-cli/manual/generate-query-help/#cmdoption-codeql-generate-query-help-format}
+   * @param outputDirectory The output directory for the generated file
+   */
+  async generateQueryHelp(pathToQhelp:string, format: string, outputDirectory?: string): Promise<string> {
```

Comment:
You might as well hardcoded `--format=markdown` in the call below, rather than allowing the format to be a parameter of this function. I don't think we need to support any other format.